integr8ly/integreatly-operator
# MGDAPI-4618 - address all linter issues with golangci-lint

# Issue link

[Linter Epic](https://issues.redhat.com/browse/MGDAPI-4593)

# What

Address all the errors from the Linter Epic: staticcheck and gosimple

# Verification steps




- Checkout this PR
- Run `make test/lint`
- Confirm there are no errors in the output
- Code review
